cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -129,8 +128,13 @@ public void select(CQL2PgJSON aCql2pgJson, String sqlFile, String cql, String ex
       assertEquals("CQL: " + cql + ", SQL: " + sql, expectedNames, actualNames);
     } catch (QueryValidationException | SQLException e) {
       logger.fine("select: " + e.getClass().getSimpleName()
-        + " for query " + cql + " : " + e.getMessage());
+          + " for query " + cql + " : " + e.getMessage());
       if (! expectedNames.startsWith("!")) {
+        int pos = e.getMessage().indexOf("Position:");
```

Comment:
```suggestion
        int pos = e.getMessage().indexOf("Position: ");
```
substring(pos + 10) skips the character after the colon. This character must be a space to avoid matching "Position:20".